src/main/java/com/example/demo/entities/ImageEntity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.entities;
+
+import jakarta.persistence.*;
+
+@Entity
+public class ImageEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Lob
+    private byte[] imageData;
```

Comment:
well, in general lobs/blobs are not quite recommended in RDBMS

Reply:
What to use instead?